sentences/de/climate_HassClimateSetTemperature.yaml

```
@@ -3,8 +3,13 @@ intents:
   HassClimateSetTemperature:
     data:
       - sentences:
-          - "<setzen>[ <area>] [die ]Temperatur auf <temperature>"
-          - "<setzen> [die ]Temperatur <von_dem> <name> auf <temperature>"
-          - "<setzen> [die ]Temperatur <name>[s] auf <temperature>"
-          - "([Die ]Temperatur|[Die ]Heizung|[(Den|Das) ]Thermostat)[ (<area>|[<von_dem> ]<name>)] auf <temperature>[ <setzen>]"
+          - "<setzen>[ <area>] <heizung> auf <temperature>"
+          - "<setzen> <heizung> <von_dem> <name> auf <temperature>"
+          - "<setzen> <heizung> <name>[s] auf <temperature>"
+          - "<heizung>[ (<area>|[<von_dem> ]<name>)] auf <temperature>[ <setzen>]"
           - "<area> auf <temperature> [be]heizen"
+          - "<setzen> <area_heizung> auf <temperature>"
+          - "<area_heizung> auf <temperature>[ <setzen>]"
+        expansion_rules:
+          heizung: ([(die|den|das) ](temperatur|heizung|thermostat|klima[anlage|tisierung]))
+          area_heizung: ([(die|den|das) ]<area><heizung>)
```

To not introduce unnecessary permutations (like `den temperatur` or `das klimatisierung`), let's use the following. I also added a ` ` between `<area>` and `<heizung>` as at least Nabu Casa TTS switches between with or without space (`Wohnzimmerheizung` and `Wohnzimmer Heizung`).

```suggestion
          heizung: ([die ]temperatur|[die ]heizung|[(das|den) ]thermostat|[die ]klima(anlage|tisierung)|[das ]klima)
          area_heizung: ([die ]<area>[ ]temperatur|[die ]<area>[ ]heizung|[(das|den) ]<area>[ ]thermostat|[die ]<area>[ ]klima(anlage|tisierung)|[das ]<area>[ ]klima)
```

---

Please also add some (not all) tests for the added space (`das wohnzimmer thermostat...`)